Dockerfile

```diff
@@ -1,3 +1,5 @@
+FROM --platform=$TARGETOS golang:alpine AS build
```

Comment:
Is this really needed?

Reply:
Yes. Without it the image build fails. Feel free to try it out and see if you can get it to work without. For me it fails without it.  Check this out https://docs.docker.com/build/building/multi-platform/

Reply:
Okay then, hmm the easiest way to test it is to merge it

Reply:
@jtkech 
You can also checkout the branch if you are using docker desktop and then run the docker build on the file locally

Reply:
@sebastienros asked to re-target 1.7 for some reason. So I want waiting his response before merging it into main to build the preview image.

Reply:
Okay no problem